shanoir-ng-front/src/app/shared/roles/auth-admin-or-expert-guard.ts

```diff
@@ -32,10 +32,9 @@ export class AuthAdminOrExpertGuard  {
         if (this.keycloakService.isUserAdminOrExpert()) {
             return true;
         } else {
-            this.router.navigate(['/home']);
             this.consoleService.log('warn', 'Sorry, you have no right to visit to this page.', ['route : ' + this.router.url]);
```

Comment:
Hi @pierrehenri-dauvergne, thank you! Please modify: 'no right to visit this page.' THX!